packages/core/src/util/config.ts

```diff
@@ -91,4 +91,8 @@ export class Config {
   static getPostHogApiKey(): string | undefined {
     return getEnvVar("POST_HOG_API_KEY");
   }
+
+  static getUseMockS3Utils(): boolean {
+    return getEnvVar("USE_MOCK_S3_UTILS") === "true";
+  }
```

Comment:
This is not part of this PR, is it?

Reply:
No ill remove

Reply:
Ill remove since its not being used anywere but seems like its already part of the code

Reply:
Sounds good!

> seems like its already part of the code

FYI, that usually happens after a bad conflict resolution.